tests/utils.go

```diff
@@ -3962,35 +3946,29 @@ func StartTCPServer(vmi *v1.VirtualMachineInstance, port int) {
 	Expect(err).ToNot(HaveOccurred())
 }
 
+func StartHTTPServerWithLoggedInVMI(vmi *v1.VirtualMachineInstance, port int, isFedoraVM bool) {
```

Comment:
@maiqueb I think you ref the wrong PR (it is closed).

If this func is left here, the `WithLoggedInVMI` text is redundant. If you think it is needed, better add it to the function doctext.

Reply:
If I remember correctly calling LoggedIn twice was not working just for Fedora and since I have go back to cirros at this PR I am going to remove WithLoggedIn, it's ok ?

Reply:
> @maiqueb I think you ref the wrong PR (it is closed).

@EdDev you're right, my bad; I meant #4129 .

Reply:
One thing I forgot,  skipping `LoggedIn` make things much faster since the ipv6 part of it is sloooow.

Reply:
I have rebased and refactored the way to start the server without calling LoggedIn functions, closing this.